CHANGELOG.md

```diff
@@ -52,7 +52,7 @@ to [sourceforge feature requests](https://sourceforge.net/p/jabref/features/) by
 - [#1023](https://github.com/JabRef/jabref/issues/492) ArXiv fetcher now also fetches based on eprint id
 
 ### Fixed
-- Fixed "Open-folder" action in Windows not opening tge correct folder when folder path contained whitespaces
+- Fixed "Open-folder" action in Windows not opening the correct folder when folder path contained whitespaces
```

Comment:
(Repetition of another comment - just to be sure, it doesn't get lost)

Please remove that line. It worked fine at the last released version 3.2. Thus, no CHANGELOG entry required.